CONTRIBUTING.md

```diff
@@ -89,11 +89,15 @@ to both `build.sbt` **and** `project/build.sbt` (so that the sbt plugin added in
 
 Releases to [Maven Central](https://repo1.maven.org/maven2/org/virtuslab/ash/) are triggered by pushing a lightweight git tag with a version number.
 
-To publish version x.y.z, type in the console
+To publish version x.y.z, type in the console (on main branch)
 ```shell
 git tag vx.y.z
 git push origin vx.y.z
 ```
 The tagged commit is then released to Maven Central.
 
 Note that we are using an early semantic versioning scheme.
```

Comment:
While you're here, pls add a link to explanation what **early** semver means (as my impression is that it's mostly used in Scala world)

Reply:
done in the newest commit